src/pages/point/Point.tsx

```diff
@@ -13,9 +13,9 @@ export const Point = () => {
   const { id } = useParams();
 
   const { isLoading, data, error } = useFetchCollection<PointType>(
-    query(getCollection("point"), where("name", "==", "Test"))
+    query(getCollection("point"), where("type", "in", ["louka", "les", "park"]))
   );
-  const pointData = data?.[0];
+  const pointData = data?.[7];
```

Comment:
takhle urcite ne, tam se bral prvni item protoze to vracelo pole. Ted to vraci objekt